docs/configuration-cluster.md

```diff
@@ -169,6 +169,11 @@ This will put additional load on the data store but will ensure that dimension a
 
 How often should source schema be reloaded in ms. Default value of 0 disables periodical source refresh.
 
+**sourceTimeBoundaryRefreshInterval** (number), minimum: 1000, default: 60000
+
+How often should source max time be refreshed in ms. 
```

Comment:
Maybe add that this sends a "timeBoundary" metadata query in druid, maybe linking the [druid docs](https://druid.apache.org/docs/latest/querying/timeboundaryquery.html) for it?